index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="menu-container">
+        <nav class="navbar">
+            <div>
+            <h1 class="logo">My Logo</h1>
+           </div>
+            <div>
+          <img src="./images/Union.png" alt="menu">
+            </div>
+        </nav>
+    </header>
+    <main class="main">
+        <h1 class="heading1">I'm Manzi Glad to see you</h1>
+        <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <h3 class="conn">LET’S CONNECT</h3>
+        <ul class="media-container">
+            <li><a href="#"><img src="./images//Vector.png"></a></li>
+            <li><a href="#"><img src="./images/Linkedin icon.png"></a></li>  
+            <li><a href="#"><img src="./images/Vector (1).png"></a></li>
+            <li><a href="#"><img src="./images/Vector (2).png"></a></li>
+            <li><a href="#"><img src="./images/Vector (3).png"></a></li> 
+          </ul>
+    </main>
```

Comment:
- [ ] Again as mentioned above, please use Flexbox for this section's layout too :+1: